nss-day-cohort-23/group-project-pinterest-the-aliases
# Val boards

Testing:

1. Click on Boards in navbar
    - five boards titled Nicolas cage should pop up
2. Go into BoardCtrl.js
3. Enter in "1" into uid variable (commented as dummy data)
4. Load page in browser
    A message should pop up saying you have no boards
5. Go to BoardCtrl.js
6. Enter in "1234" in uid variable
7. Load page in browser
    -boards should pop up
8. Type in search bar to filter through board titles
9. Click on  a board
     It should direct you to the single bard view. Board id is in address bar
